internal/terraform/node_action_trigger_instance_plan.go

```diff
@@ -89,7 +89,8 @@ func (n *nodeActionTriggerPlanInstance) Execute(ctx EvalContext, operation walkO
 	}
 
 	if change == nil {
-		panic("change cannot be nil")
+		// nothing to do (this may be a refresh )
+		return nil
```

Comment:
We might want something more robust here eventually, having a `nil` change under any other circumstance is worthy of a panic (and with the refactoring that will probably continue around actions, mistakes like a missing change are even more likely)

Reply:
Perhaps we shouldn't have even gotten to this point during a refresh only plan, since `actions` are in effect changes, and we're promising that there can be no changes with `-refresh-only`.